eng/build.ps1

```diff
@@ -24,7 +24,7 @@ $possibleDirToBuild = if($ExtraArgs.Length -gt 0) { $ExtraArgs[0]; } else { $nul
 if ($possibleDirToBuild -ne $null) {
   $dtb = $possibleDirToBuild
   if (Test-Path $dtb) {
-    $ExtraArgs[0] = "/p:DirectoryToBuild=$(Resolve-Path $dtb)"
+    $ExtraArgs[0] = "/p:DirectoryToBuild=$(Resolve-Path $dtb.TrimEnd('\'))"
```

Comment:
Should we trim possibleDirToBuild instead? Will this not fail in the else statement as well?

Reply:
No the else statement is only dealing with project names, i.e. `build System.Runtime`

Reply:
it could also apply to `build System.Runtime\tests\` (for lazy people that don't want to type in `src\System.Runtime\tests\`

Reply:
I didn't know that this scenario is even supported. Just tried it locally, you are absolutely right. Will change.

Reply:
done